settings.gradle

```diff
@@ -34,7 +34,9 @@ include ':integration_tests:androidx_test'
 include ':integration_tests:ctesque'
 include ':integration_tests:security-providers'
 include ":integration_tests:mockk"
-include ":integration_tests:jacoco-on-the-fly"
+include ':integration_tests:jacoco'
```

Comment:
Three projects to test Jacoco seems a little excessive. Don't we just need to test jacoco-offline, i.e. where Robolectric attempts to instrument classes that are already instrumented by Jacoco?

Reply:
Yeah, it did.
Sorry for that I'm trying to add all JaCoCo stuffs in one go.

How about adding jacoco-offline tests and removing existed jacoco-on-the-fly tests?

I can submit the  jacoco-on-the-fly tests later when we need to verify the existed corporation problems(coverage 0) with JaCoCo. And submit a nicer optimization later on all jacoco tests are completed.

Actually, the `:integration_tests:jacoco` project is just a parent project, much more like a container that holds two subprojects and common main/test codes.
It can be replace it with a directory instead of a project container.

Reply:
That sounds good, I think we only need one Jacoco project. Whether it's on-the-fly or offline doesn't matter much, I think they are equivalent. We just need to test robolectric instrumentation on jacoco-instrumented classes.

Reply:
@ZSmallX I remember the internal problem only appears with offline JaCoCo mode, what about keeping the offline mode only?

Reply:
Yeah 👍 We should only have the offline tests for now. With on-the-fly-tests, jacoco is instrumented classes that have already been instrumented by Robolectric, which has not been an issue.